apps/omar-ui-app/grails-app/controllers/omar/app/OmarController.groovy

```diff
@@ -10,39 +10,26 @@ import omar.ui.OmarSitesConfig
 class OmarController /*implements InitializingBean*/
 {
   def openlayers
+  def userInfo
 
   OpenLayersConfig openLayersConfig
   OmarSitesConfig omarSitesConfig
 
   def index()
   {
-      // println openLayersConfig as JSON
-      // println grailsApplication.config.omar.openlayers as JSON
 
-  		// def clientConfig = new ConfigObject()
-  		// clientConfig.params = grailsApplication.config.omar.app
-      //
-  		// clientConfig.openlayers = OmarOpenlayersUtils.openlayersConfig
-      // // clientConfig.openlayers = openlayers
-      //
-  		//  // Use Enhancer traits from omar-core getBaseUrl()
-  		// clientConfig.serverURL = getBaseUrl()
-      //
-  		// // Params to pass to client
-  		// def clientParams = grailsApplication.config.omar.app
+     userInfo = grailsApplication.config.omar.app.userInfo
+     println 'requestHeaderUserName: ' + userInfo
```

Comment:
Should this print be a log?

Reply:
Plus,  try to avoid doing string concat.   Use string interp instead.

Reply:
Removed.  See commit below.